sale_order_product_recommendation_packaging_default/wizards/sale_order_recommendation_view.xml

```diff
@@ -36,12 +44,13 @@
                     name="product_packaging_id"
                     groups="product.group_stock_packaging"
                     context="{'default_product_id': product_id}"
+                    attrs="{'required': [('require_packaging', '=', True)]}"
                 />
                 <field
                     name="product_packaging_qty"
                     groups="product.group_stock_packaging"
                     widget="numeric_step"
-                    attrs="{'readonly': [('product_packaging_id', '=', False)]}"
+                    attrs="{'readonly': ['|', ('product_packaging_id', '=', False), ('require_packaging', '=', True)]}"
                 />
```

Comment:
issue: just like below, this field should be writable always when there's a packaging. But instead, do the opposite and make the `product_uom_qty` field readonly when `require_packaging == True`.